src/lru_cache.rs

```diff
@@ -343,19 +236,13 @@ impl<A: fmt::Show + Hash + Eq, B: fmt::Show> fmt::Show for LruCache<A, B> {
     /// used to least-recently used.
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
No, because `LruCache` uses reverse iterator in `fmt`, and `LinkedHashMap` uses forward iterator.